static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -66,14 +64,14 @@ export const getUserFromCookie = () => {
 };
 
 export const updateNewsletter = (newsletter) =>
-    reqwest({
-        url: `${config.get('page.idApiUrl')}/users/me/newsletters`,
+    fetchJson(`${config.get('page.idApiUrl')}/users/me/newsletters`, {
         method: 'PATCH',
-        type: 'json',
-        contentType: 'application/json',
-        withCredentials: true,
-        crossOrigin: true,
-        data: JSON.stringify(newsletter),
+        headers: new Headers({
```

Comment:
What does `new Headers` do here? I always thought you could pass a simple object for `headers`

Reply:
> **`headers`**
Any headers you want to add to your request, contained within a [`Headers`](https://developer.mozilla.org/en-US/docs/Web/API/Headers) object or an object literal with [`ByteString`](https://developer.mozilla.org/en-US/docs/Web/API/ByteString) values. Note that [some names are forbidden](https://developer.mozilla.org/en-US/docs/Glossary/Forbidden_header_name).

You’re right, I just got carried away with _typing_. I thought it was cleaner/nicer, but happy to keep it as an object literal.

Reply:
Oh, there's a Headers object? I learn something new every day around here!

I'm utterly cool with either way, I don't think it matters too much so whatever you think is best is fine with me.

Reply:
There’s the added complexity of having to import in in node, so reverted to an object literal in 2781ed4